notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -405,18 +407,8 @@
     }
    },
    "source": [
-    "### Run training pipeline\n",
-    "Tune floating-point ResNet-18 on Tiny-ImageNet-200 to obtain a pre-trained model for quantization."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "118rlV22_PB5"
-   },
-   "source": [
-    "> **NOTE** The model is not tuned till the final accuracy. For the sake of simplicity of the tutorial, we tune for 4 epoch only and take the last model state for quantization.\n",
-    "\n"
+    "### Get a pretrained FP32 model\n",
```

Comment:
```suggestion
    "### Get a pre-trained FP32 model\n",
```
In the text below you write pre-trained, so for consistency I suggest adding the hyphen here too.